cmd/ocm-backplane/login/login.go

```diff
@@ -170,9 +172,9 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 	targetCluster.Server = bpAPIClusterUrl
 
 	// Add proxy URL to target cluster
-	globalOpts.ProxyURL = bpConfig.ProxyURL
```

Comment:
I guess it's necessary to update proxy URL if it's not specified via `--proxy`
```
if globalOpts.ProxyURL == "" {
   globalOpts.ProxyURL = bpConfig.ProxyURL
}
```

Reply:
It picks up from the config file if it isn't specified explicitly

Reply:
nice :+1: